res/values-hr/strings.xml

```diff
@@ -1254,70 +1254,70 @@
     <string name="Battle_Royale">Battle Royale</string>
     <string name="The_winner_will_receive_x_plasma_">Pobjednik će dobiti %s plazme !</string>
     <string name="admin_commands">"
-Commands:
-!kick [0–31]
-!reset
-!welcome {message}
-!difficulty [0-3] or [1-4] for Campaign
-!hidden
-!guests
-!slowchat
-!pausechat
-!mute {id}
+Komande:
```

Comment:
Okay, will do.

Reply:
I left an explanation in commit but on my phone idk why everything messed up if i tried to add descriptions on it, so I just put it in commit description so you can see it if you need it

Reply:
Ive checked the whole page again, so I dont make another commit, can you correct where its "spectate killer on death" the word "Ubijo" just remove letter J please because its the only grammar mistake ive made that's why I triple checked

Reply:
Check the latest commit, I fixed grammar mistakes

Reply:
By design i can't edit my original commit without making new ones, will you check everything ive fixed and put the edits yourself?